src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -49,7 +49,12 @@ public static List<PartyDetails> buildHearingPartiesDetails(HearingWrapper wrapp
         }
 
         if (isYes(caseData.getJointParty().getHasJointParty())) {
-            partiesDetails.add(createJointPartyDetails(caseData));
+            partiesDetails.addAll(
+                buildHearingPartiesPartyDetails(
+                    caseData.getJointParty(),
+                    appeal.getHearingOptions(),
```

Comment:
I don't think you shouldn't be using the appellant's hearing options for a joint party

Reply:
Actually Unit testcases are failing for null point exception for hearingoptions